core/src/main/java/de/jplag/GreedyStringTiling.java

```diff
@@ -219,33 +216,14 @@ private boolean[] calculateInitiallyMarked(Submission submission) {
 
     private SubsequenceHashLookupTable subsequenceHashLookupTableForSubmission(Submission submission, boolean[] marked) {
         return cachedHashLookupTables.computeIfAbsent(submission,
-                (key -> new SubsequenceHashLookupTable(minimumMatchLength, tokenValueListFromSubmission(key), marked)));
-    }
-
-    /**
-     * Converts the tokens of the submission to a list of values.
-     * @param submission The submission from which to convert the tokens.
-     */
```

Comment:
Your are removing documention here (by replacing documented code with undocumented code). This is often a giveaway that newly written code should be documented.